browser/mapping_test.go

```diff
@@ -158,11 +186,17 @@ func toFirstLetterLower(s string) string {
 	}
 }
 
-// isWildcard returns true if the method is a wildcard method and
-// returns the name of the method to be called instead of the original
-// method.
-func isWildcard(wildcards map[string]string, typ, method string) (string, bool) {
+// isWildcardOrGetter returns true if the method is a wildcard or getter method
+// and returns the name of the method to be called instead of the original one.
+func isWildcardOrGetter(wildcards, getters map[string]string, typ, method string) (string, bool) {
 	name := typ + "." + method
-	s, ok := wildcards[name]
-	return s, ok
+
+	if s, ok := wildcards[name]; ok {
+		return s, ok
+	}
+	if s, ok := getters[name]; ok {
+		return s, ok
+	}
+
+	return "", false
```

Comment:
```suggestion
// isCustomMapping returns true if the method is a custom mapping
// and returns the name of the method to be called instead of the
// original one.
func isCustomMapping(customMappings map[string]string, typ, method string) (string, bool) {
	name := typ + "." + method

	if s, ok := customMappings[name]; ok {
		return s, ok
	}

	return "", false
}
```